test/automation/src/positron/positronConsole.ts

```diff
@@ -118,4 +121,54 @@ export class PositronConsole {
 	async maximizeConsole() {
 		await this.code.waitAndClick(MAXIMIZE_CONSOLE);
 	}
+
+	async pasteCodeToConsole(code: string) {
+		const activeConsole = await this.getActiveConsole();
+		const consoleInput = activeConsole?.locator(CONSOLE_INPUT);
+		await this.pasteInMonaco(consoleInput!, code);
+	}
+
+	// adapted from:
+	// https://github.com/deephaven/web-client-ui/blob/9d905fca86aa8ba4ff53debd1fd12dcc9132299b/tests/utils.ts#L107
+	async pasteInMonaco(
+		locator: Locator,
+		text: string
+	): Promise<void> {
+		const page = locator.page();
+		const isMac = os.platform() === 'darwin';
+		const modifier = isMac ? 'Meta' : 'Control';
+
+		// Create a hidden textarea with the contents to paste
+		const inputId = await page.evaluate(async evalText => {
+			const tempInput = document.createElement('textarea');
+			tempInput.id = 'super-secret-temp-input-id';
+			tempInput.value = evalText;
+			tempInput.style.width = '0';
+			tempInput.style.height = '0';
+			document.body.appendChild(tempInput);
+			tempInput.select();
+			return tempInput.id;
+		}, text);
+
+		// Copy the contents of the textarea which was selected above
+		await page.keyboard.press(`${modifier}+C`);
+
+		// Remove the textarea
+		await page.evaluate(id => {
+			document.getElementById(id)?.remove();
+		}, inputId);
+
+		// Focus monaco
+		await locator.click();
```

Comment:
I removed everything but the `DataTransfer()` line and the test still succeeded.

Reply:
(This comment doesn't make much sense outside of the "Files Changed" view on github.)

Reply:
By chance did you run once with that code in place, then remove it and run again with just the data transfer?  I am guessing this because if that was the flow you would already have the stuff to paste inside your clipboard so you wouldn't need to re-add it, if that makes sense.

Reply:
oh wait, I see what you mean.  let me remove that stuff and retest.  I think those extra steps may have been for firefox, but then i removed the firefox stuff.

Reply:
Great observation.  I had a bunch of complicated dead code.  Cleaned up all the dead stuff.

Thanks!